iabulkupload/README.md

```diff
@@ -0,0 +1,23 @@
+# iabulkupload
+The tools in this folder can be used for bulk upload of Thoth publishers' back catalogues to Internet Archive via the Thoth Dissemination Service.
+This README records the steps taken to upload the OBP and punctum back catalogues to the [Thoth Archiving Network collection](https://archive.org/details/thoth-archiving-network) on 2022-11-28/29.
+See also [the README for the Thoth Dissemination Service itself](https://github.com/thoth-pub/thoth-dissemination/blob/v0.1.0/README.md).
+
+### Steps to upload
+1. Check out clean version of Thoth Dissemination Service v0.1.0 to parent folder `thoth-dissemination`.
+2. Ensure that the appropriate Internet Archive credentials are present in `../config.env`.
```

Comment:
could clarify that crendentials can either be present in the config file or in the environment

Reply:
I couldn't find a way to make the alternative method work (maybe because of https://github.com/thoth-pub/thoth-dissemination/blob/1e3106ec58c888f5c4d70eccb02a75f8c14e3e28/disseminator.py#L77?), so to avoid confusion I haven't added any reference to it.

Reply:
What's the alternative method, setting the variables in the environment? If so, that's what dotenv does

Reply:
Yes - I tried doing `export ia_s3_access=[etc]` in place of this step, but then when I ran the disseminator in the Docker image it didn't seem to pick them up. Is there a different way I should be doing it?

Reply:
Well, if it's run with docker then you need to pass them as `--env export ia_s3_access=[etc]`. But with the undockerised version you should just be able to `export` them; were you using a virtualenv or something else to run it?

Reply:
Ah, I assumed you meant this could be done without changing `bulkupload.sh`, since the later steps rely on it. I can write something to explain that option, in that case.